diff --git a/Makefile.am b/Makefile.am
index d9d6fed..ed2d7bf 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = . test
+SUBDIRS = . loleaflet 
 
 export ENABLE_DEBUG
 
diff --git a/loleaflet/Makefile b/loleaflet/Makefile
index ae9803b..663d34f 100644
--- a/loleaflet/Makefile
+++ b/loleaflet/Makefile
@@ -32,6 +32,12 @@ build-l10n:
 
 all: build build-l10n
 
+.PHONY: install
+install: dist workaround-install
+workaround-install:
+	mkdir -p ../../install/usr/share/loolwsd/loleaflet 
+	cp -ar dist ../../install/usr/share/loolwsd/loleaflet
+
 .PHONY: dist
 dist: MINIFY=true
 dist: all
