diff --git a/wsd/LOOLWSD.cpp b/wsd/LOOLWSD.cpp
index edc463a..9b52ba0 100644
--- a/wsd/LOOLWSD.cpp
+++ b/wsd/LOOLWSD.cpp
@@ -1522,10 +1522,10 @@ LOOLWSD::~LOOLWSD()
 
 void LOOLWSD::initialize(Application& self)
 {
-    if (geteuid() == 0)
+    /*if (geteuid() == 0)
     {
         throw std::runtime_error("Do not run as root. Please run as lool user.");
-    }
+    }*/
 
     if (!UnitWSD::init(UnitWSD::UnitType::TYPE_WSD, UnitTestLibrary))
     {
