diff --git a/common/Common.hpp b/common/Common.hpp
index a0a7568..ae60f69 100644
--- a/common/Common.hpp
+++ b/common/Common.hpp
@@ -18,7 +18,7 @@ constexpr int DEFAULT_CLIENT_PORT_NUMBER = 9980;
 constexpr int DEFAULT_MASTER_PORT_NUMBER = 9981;
 
 constexpr int COMMAND_TIMEOUT_MS = 5000;
-constexpr long CHILD_TIMEOUT_MS = COMMAND_TIMEOUT_MS;
+constexpr long CHILD_TIMEOUT_MS = 30000;
 constexpr int CHILD_REBALANCE_INTERVAL_MS = CHILD_TIMEOUT_MS / 10;
 constexpr int POLL_TIMEOUT_MS = COMMAND_TIMEOUT_MS / 10;
 constexpr int WS_SEND_TIMEOUT_MS = 1000;
